tests/auth_tests/test_hashers.py

```diff
@@ -466,7 +466,9 @@ def test_harden_runtime(self):
             self.hasher.harden_runtime('password', 'encoded')
 
     def test_must_update(self):
-        self.assertIs(self.hasher.must_update('encoded'), False)
+        msgregex = r'Method `decode` not fully implemented for .*. Decode will be required in future releases.'
```

Comment:
This can go away now, but needs a new test that the warning is raised in the base hashers `decode`